test/autorest/generated/headergroup/internal/headergroup/header.go

```diff
@@ -358,11 +358,11 @@ func (HeaderOperations) ResponseDatetimeHandleResponse(resp *azcore.Response) (*
 	if !resp.HasStatusCode(http.StatusOK) {
 		return nil, newError(resp)
 	}
-	val, err := time.Parse(time.RFC3339, resp.Header.Get("value"))
+	Value, err := time.Parse(time.RFC3339, resp.Header.Get("value"))
```

Comment:
I think this should be `timeRFC3339Nano`, can be fixed separately.

Reply:
Adding a separate issue to track this fix: #298